speech/google/cloud/speech/streaming/container.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2016 Google Inc.
```

Comment:
I am :-1: on `google.cloud.speech.streaming` package. Just put these in the `google.cloud.speech` package